apriori/slow-games.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Slow Games
+category: distributed systems
+publish_date: 2024-08-21
+image: media/slowgamescover.png
+excerpt: This article is a summary of *Slow Games; Policy Enforcement under Uncertainty*. We explore the slow game by reviewing a conceptual framework, lossy compression model, an example game model, simulations, and future directions. 
+---
+## Preface
+This art is a summary based on the Anoma Research Topic titled *Slow Games: Policy Enforcement under Uncertainty* by D Reusche, Christopher Goes and Nicolas Della Penna. All credit to the authors. Thank you to D for feedback and review. Any errors *are* my own.
+
+## Introduction
+The Slow Game is a type of principal-agent problem. The principal-agent problem ([PAP](https://en.wikipedia.org/wiki/Principal%E2%80%93agent_problem)) arises when one entity takes actions on behalf of another entity, where there is a conflict of interest. 
+
+There are two actors in the slow game. 
+- Operator - operator of a service. This is the agent. 
+- Regulator - attempts to enforce policies of the service being provided by the operator. This is the principal.
+
+The regulator is slower to act than the operator, and in some cases less sophisticated. Critically, the regulator is slower in measuring the operator's actions. This introduces uncertainty depending on the difference in speed between the two actors. 
```

Comment:
I would include some examples of Slow Games here, e.g.:
- validator set selection
- solver selection